v3/src/components/graph/utilities/pixi-points.ts

```diff
@@ -470,6 +501,14 @@ export class PixiPoints {
     for (let i = 0; i < oldPointsCount; i++) {
       const point = this.points[i]
       if (point.texture !== texture) {
+        // The anchor should be set according to the point's shape.
+        // Circle: center (0.5)
+        // Horizontal Bar: bottom left corner (1,0)
+        // Vertical Bar: top left corner (0,0)
+        point.anchor.set(
+          this.displayType === "bars" ? (this.barOrientation === "horizontal" ? 1 : 0) : 0.5,
+          this.displayType === "bars" ? 0 : 0.5
+        )
```

Comment:
On slack you asked whether there might be a way to avoid passing the `barOrientation` into this class. It's not a huge win, but you could pass the `anchor` in instead of the `barOrientation`. You could define `anchor` constants somewhere, e.g.
```typescript
// The anchor should be set according to the point's shape.
export const circleAnchor: PIXI.IPointData = { x: 0.5, y: 0.5 }
export const hBarAnchor: PIXI.IPointData = { x: 1, y: 0 }
export const vBarAnchor: PIXI.IPointData = { x: 0, y: 0 }
```
and then the client would pass the appropriate `anchor` value and this code would become:
```suggestion
        point.anchor.copyFrom(this.anchor)
```
This might make it easier to extend support to other shapes, for instance.

Reply:
I like this idea as a small improvement. In practice though I couldn't get it to work. Even though I set and updated an `anchor` property in essentially all the same places I set and updated the `barOrientation` property, `PixiPoints` would not honor the updated anchor value when rendering bars. I'm not sure why. I do want to work that out, but if it's OK with you, I'd prefer to let it go for this PR and figure it out in the next PR for getting the bars to appear in a smooth transition.